src/themes/dark.tsx

```diff
@@ -19,6 +19,12 @@ const darkTheme = {
   heroImageBackgroundColor: "#2f363d !important",
   subtitleColor: colors.gray[4],
   mapBackgroundColors: {
+    "New Queen Street": "#27dfdf",
+    Esperança: "#d3b8a7",
+    Colosseo:"#e79b68",
+    Paraíso:"#41ce4c",
```

Comment:
```suggestion
    Paraíso: "#41ce4c",
```
![image](https://user-images.githubusercontent.com/82317/201140336-11ebf009-2534-433c-a137-b157dc1d523c.png)